app/views/events/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="row">
+<aside class="col-md-12">
+<section class="user_info">
+
+<h3>Upcoming Events (<%= @upcoming_events.count %>)<span><%= link_to "Create new", new_event_path %></span></h3>
+<% if @upcoming_events.any? %>
+  <div class="events">
+      <% @upcoming_events.each do |event| %>
+      <div class="event">
+          <span class="name"><%= link_to event.name, event_path(event)%></span>
+          <span><%= event.creator.name %></span>
+          <span class="description"><%= event.description %></span>
+          <span class="dateofevent">Date: <%= event.dateofevent %></span>
+          <span class="location">Location: <%= event.location %></span>
+
+
+          <span class="timestamp">
+              Posted <%= time_ago_in_words(event.created_at) %> ago.
+          </span>
+      </div>
+      <% end %>
+  </div>
+<% end %>
+
+<h3>Previous Events (<%= @previous_events.count %>)</h3>
+<% if @previous_events.any? %>
+<div class="events">
+    <% @previous_events.each do |previous_event| %>
```

Comment:
You could have used a standard `event` partial to render events whether they be previous or upcoming events. No need for an abuse of `each`.